modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -807,7 +807,7 @@ CALL unsupported.dbms.shutdown
 ----
 
 Description of the returned code::
-The query used a deprecated procedure. (`'unsupported.dbms.shutdown'` is no longer supported)
+The query used a deprecated procedure: `'unsupported.dbms.shutdown'`.
```

Comment:
I guess this one is actually unsupported 😄 But that you see in the name of the procedure ^^